src/code/z_en_item00.c

```diff
@@ -739,7 +739,7 @@ void EnItem00_DrawRupee(EnItem00* this, PlayState* play) {
 }
 
 TexturePtr D_801ADF44[12] = {
-    gameplay_keep_Tex_05E6F0, // Heart (Not used)
+    gEffHeartTex,             // Heart (Not used)
```

Comment:
```suggestion
    gDropRecoveryHeartTex,             // Heart (Not used)
```
This is the name in OoT and it's identical, so it should stay consistent. Plus, the `Eff` prefix is mainly reserved for the "effects" system, which this texture is not a part of